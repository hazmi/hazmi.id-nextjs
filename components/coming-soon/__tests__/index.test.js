import React from 'react'
import renderer from 'react-test-renderer'
import ComingSoon from '../index'

describe('With Snapshot Testing', () => {
  it('show the correct content', () => {
    const component = renderer.create(<ComingSoon />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})