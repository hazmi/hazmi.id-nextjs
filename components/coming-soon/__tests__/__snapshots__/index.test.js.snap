// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing show the correct content 1`] = `
<section
  className="coming-soon"
>
  <p>
     is coming soon!
  </p>
</section>
`;
